bittensor/_neuron/text/core_validator/__init__.py

```diff
@@ -299,6 +300,11 @@ def __enter__(self):
                 root_dir = self.config.neuron.full_path
             )
 
+        # Check if no_serve is set.
+        if self.config.neuron.no_serve:
+            print('--neuron.no_serve flag set, not serving to network. Prometheus metrics will not be available.')
```

Comment:
Rather than printing a simple string, this should be a warning issued by the logger, `logger.warning`